### ISSUE TITLE
 Custom UI Control in RadListView with DataBinding - Property set is called twice 

### ISSUE THREAD

**Issue by matecode** [telerik/nativescript-ui-feedback#736]
Hi. I’m new to NativeScript. While I’m discovering a lot, i found a problem i cannot solve on my own, so i’m hopping to find some help. Maybe I'm doing something wrong (please tell me what) or maybe it's a bug, we'll see.

### Did you verify this is a real problem by searching the [NativeScript Forum]
Yes ... i didn't found anything yet

### Tell us about the problem
Therefor i created a playground to easily follow my steps:
https://play.nativescript.org/?template=play-js&id=lhZIAv

I have a viewmodel with an array (items) that i want to show as a gridlist. So i use a RadListView. As item i want to use a custom Ui control which I created (Item). This Item has also a property called “item” (Sorry for naming). In xml of home-page i pass the {{$value}}. For better understanding i put a console.log to the set method of the property.

So whats happening: The set method is called twice for every item: first with the whole ViewModel-Object (i guess) and secondly with the real entry from the Array.

### Which platform(s) does your issue occur on?
I only tested this on Android by now. API 26

### Please provide the following version numbers that your issue occurs with:
( Also happening on Playground wher i cant see the versions )
- **Progress NativeScript UI version:** ^3.5.9
- **CLI:** 4.1.0
- **Cross-platform modules:** ^4.1.0
- **Runtime(s):** "tns-android" 4.1.3

### Please tell us how to recreate the issue in as much detail as possible.
https://play.nativescript.org/?template=play-js&id=lhZIAv
1. Start the application ..
2. See Log....

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
https://play.nativescript.org/?template=play-js&id=lhZIAv

Thanks for helping me.

**Comment by NickIliev** [comment]
The issue seems to be reproducible with `RadListView` only and not with `L:istView` - sample project [here](https://play.nativescript.org/?template=play-js&id=lhZIAv&v=4)

**Comment by matecode** [comment]
@NickIliev thats true, i also tried with listview asnd all was fine, but i wanted to use a gridlist
